### PR TITLE
add eye tracker tags

### PR DESCRIPTION
**Pull request by marcobarilari**
to be tested with the eyetracker if it actually work but you can review the 'code style'

### CODE REVIEW

File: visualLocTranslational.m [hunk 1]
```diff
@@ -72,11 +75,13 @@
 
         fprintf('\n - Running Block %.0f \n', iBlock);
 
-        eyeTracker('StartRecording', cfg);
+        eyeTracker('Message', cfg, ['start_block-', iBlock]);
```
Comment (Remi-Gau) [inline visualLocTranslational.m:78]:
```suggestion
        eyeTracker('Message', cfg, ['start_block-', num2str(iBlock)]);
```

File: visualLocTranslational.m [hunk 2]
```diff
@@ -72,11 +75,13 @@
...
 
         % For each event in the block
         for iEvent = 1:cfg.design.nbEventsPerBlock
 
+            eyeTracker('Message', cfg, ['start_trial-', iEvent, '_', thisEvent]);
```
Comment (Remi-Gau) [inline visualLocTranslational.m:83]: I think you will have a problem there, no?

iEvent is an `integer` and `thisEvent` is a structure so concatenating those will throw an error, no?

File: visualLocTranslational.m [hunk 3]
```diff
@@ -115,7 +120,7 @@
 
         end
 
-        eyeTracker('StopRecordings', cfg);
+        eyeTracker('Message', cfg, ['end_trial-', iEvent, '_', thisEvent]);
```
Comment (Remi-Gau) [inline visualLocTranslational.m:123]: same as above

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
I suspect that some those things might not work. 

Let's check again after you test it tomorrow.

File: visualLocTranslational.m (hunk 1 above)
Comment (marcobarilari) [reply visualLocTranslational.m:78]: yes, good point

**Comment by marcobarilari** [comment]
definitely, I was too fast to call the game over. Now it should be fine

**Comment by marcobarilari** [comment]
an extra thing that I would ask you to double-check is whether those messages are in the correct place in the script (exp-time and code congruency)

**Comment by marcobarilari** [comment]
cpp-lln-lab/CPP_PTB#84 is linked

**Review by Remi-Gau — APPROVED** [review]
Good to go.

Merge whenever you want.

**Pull request merged**